scripts/validate-nimbus-file.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+TARGET_PATH=/tmp/nimbus-cli
+NIMBUS_INSTALL_REMOTE_PATH=https://raw.githubusercontent.com/mozilla/application-services/main/install-nimbus-cli.sh
+NIMBUS_INSTALL_LOCAL_PATH=$TARGET_PATH/install.sh
+
+if [ -d "${TARGET_PATH}" ]; then
+  echo "Directory for \`nimbus-cli\` already exists: $TARGET_PATH"
+else
+  echo "Create directory for \`nimbus-cli\`: $TARGET_PATH"
+  mkdir -p $TARGET_PATH
+fi
+
+if [ -f "${NIMBUS_INSTALL_LOCAL_PATH}" ]; then
+  echo "Skip download of \`nimbus-cli\` installer"
+else
+  echo "Download \`nimbus-cli\` installer from $NIMBUS_INSTALL_REMOTE_PATH to $TARGET_PATH"
+  curl --proto '=https' --tlsv1.2 -sSf -o "$NIMBUS_INSTALL_LOCAL_PATH" "$NIMBUS_INSTALL_REMOTE_PATH"
+fi
```

Comment:
I would move all of this inside the else branch below

Reply:
Moved in [dcc316b](https://github.com/mozilla/blurts-server/pull/4861/commits/dcc316be86feb7bb2a3981e8a1b382b3fd88aa21).